google/cloud/bigtable/table.py

```diff
@@ -1130,11 +1143,23 @@ def _do_mutate_retryable_rows(self):
                 retry=None,
                 **kwargs
             )
-        except RETRYABLE_MUTATION_ERRORS:
+        except RETRYABLE_MUTATION_ERRORS as exc:
```

Comment:
It should also be retried in read rows

Reply:
I've updated `row_data` module to also handle rst stream retry read rows.